index.html

```diff
@@ -1,10 +1,224 @@
+          <div class="slider-buttons">
+            <button class="button">explore</button>
+            <button class="learn-btn">
+              learn more
+            </button>
+          </div>
+          <div class="slider-dots">
+            <div class="slider-dots-item slider-dots-item-focused"></div>
+            <div class="slider-dots-item"></div>
+            <div class="slider-dots-item"></div>
+          </div>
+        </div>
+      </div>
+      <div class="paralax-section">
+        <div class="paralax-headline">
+          <div class="underline">
+            <h2 class="underline-headline">Paralax section</h2>
+          </div>
+          <p class="underline-text">
+            Exploring the unexplored is a risky business, but a rewarding one.
+          </p>
+        </div>
+        <div class="paralax-quote">
+          <span class="quote-symbol">‘‘</span>
+          <p class="quote">
+            Design is in everything we make, but it's also between those things.
+            It's a mix of craft, science, storytelling, propaganda, and
+            philosophy.
+          </p>
+          <p class="quote-author">— Erik Adigard</p>
+        </div>
+        <div class="bottom-picture"></div>
+      </div>
+      <div class="commercial-data">
+        <ul class="list-of-data">
+          <li class="list-of-data-li">
+            <div class="chapter">
+              <div class="chapter-container"><img src="src/img/user.png" alt="" class="chapter-image"/>
+              <p class="list-numbers">1000+</p></div>
+              <p class="list-letters">Happy users</p>
+            </div>
+          </li>
+          <li class="list-of-data-li">
+            <div class="chapter-two">
+                <div class="chapter-container-two">
+              <img src="src/img/cart.png" alt="" class="chapter-image-two" />
+              <p class="list-numbers">200</p></div>
+              <p class="list-letters-two">Sold products</p>
+            </div>
+          </li>
+          <li class="list-of-data-li-special">
+            <div class="chapter-three">
+                <div class="chapter-container-three">
+              <img src="src/img/comments.png" alt="" class="chapter-image-three" />
+              <p class="list-numbers">5632</p></div>
+              <p class="list-letters-three">Comments</p>
+            </div>
+          </li>
+          <li class="list-of-data-li">
+            <div class="chapter-four">
+                <div class="chapter-container-four">
+              <img src="src/img/files.png" alt="" class="chapter-image-four" />
+              <p class="list-numbers">25000+</p></div>
+              <p class="list-letters-four">Lines of code</p>
+            </div>
+          </li>
+        </ul>
+      </div>
+      <div class="latest-blog-posts">
+        <div class="blog-headline">
+          <div class="blog-underline">
+            <h2 class="blog-underline-headline">Latest Blog Posts</h2>
+          </div>
```

Comment:
Why did you add the text inside `blog-underline` block? 
Styles for these elements could be easier